packages/project-account-switcher/src/presenters/ProjectAccountSwitcherPresenter.js

```diff
@@ -0,0 +1,224 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import Flyout from "@hig/flyout";
+import "@hig/flyout/build/index.css";
+import icons from "../../../icons/build";
+import { anchorPoints } from "../../../flyout/src/anchorPoints";
+import "./ProjectAccountSwitcherPresenter.scss";
+
+export default class ProjectAccountSwitcherPresenter extends Component {
+  static constructPlaceholder(label) {
+    return label.match(/\b(\w)/g).join("");
```

Comment:
Not a big deal but I believe you can omit the quotes here

Reply:
Removing the quotes results in:

<img width="257" alt="screen shot 2018-08-17 at 10 31 31 am" src="https://user-images.githubusercontent.com/9288982/44271676-c39bb600-a208-11e8-94ed-79b7dd976a68.png">

Reply:
Ah, ok not a big deal either way